builder/Dockerfile

```diff
@@ -10,6 +10,6 @@ WORKDIR /src
 RUN ["apk", "update"]
 RUN until apk add docker; do :; done
 
-RUN ["cabal", "update"]
+RUN ["stack", "upgrade"]
```

Comment:
@mbj do you think I should remove this altogether? It probably won't break anything existing.